terra_notebook_utils/samtools.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+import subprocess
+
+def _run(cmd: list, **kwargs):
+    p = subprocess.run(cmd, **kwargs)
+    p.check_returncode()
+    return p
+
+bin_root = os.path.abspath(os.path.dirname(sys.executable))
+htsfile_path = os.path.join(bin_root, "htsfile")
+bcftools_path = os.path.join(bin_root, "bcftools")
+bcftools_available = os.path.isfile(bcftools_path)
+
+try:
+    p = _run([htsfile_path, "--version"], capture_output=True)
+    htsfile_available = True
+except (FileNotFoundError, subprocess.CalledProcessError):
```

Comment:
Maybe something like:

```
except (FileNotFoundError, subprocess.CalledProcessError) as e:
    print(e)  # or traceback.print_exc()
    htsfile_available = False
```

Reply:
An exception here indicates htlib failed to build during `setup.py`. Printing out the trace would not help diagnose that problem, and would therefore be confusing.

We could do something like:
```
except (FileNotFoundError, subprocess.CalledProcessError) as e:
    print("htsfile unavailable: htslib build failed during installation. try `pip install -v terra-notebook-utils` to diagnose the problem")
    htsfile_available = False
```

Reply:
done.